pkg/generator/function.go

```diff
@@ -55,10 +55,15 @@ func (g *Generator) FuncToString(f *model.Function) string {
 	}
 
 	for _, args := range f.AdditionalArgs {
+		fullType := args.FullType()
+		if strings.Contains(args.Type, "/") {
+			re := regexp.MustCompile(`^([^a-zA-Z]*)([a-zA-Z].*/)(.+)$`)
```

Comment:
According to my research:
- A domain name label consists of letters, digits, and hyphens
- It can start with a number, but not with hyphens

So, it'd be better to include numbers for future forked implementations.
```suggestion
			re := regexp.MustCompile(`^([^a-zA-Z0-9]*)([a-zA-Z0-9].*/)(.+)$`)
```